src/uucore/src/lib/features/signals.rs

```diff
@@ -387,6 +388,25 @@ pub fn ignore_interrupts() -> Result<(), Errno> {
     unsafe { signal(SIGINT, SigIgn) }.map(|_| ())
 }
 
+// Disables the custom signal handlers installed by Rust for stack-overflow handling. With those custom signal handlers processes ignore the first SIGBUS and SIGSEGV signal they receive.
+// See https://github.com/rust-lang/rust/blob/8ac1525e091d3db28e67adcbbd6db1e1deaa37fb/src/libstd/sys/unix/stack_overflow.rs#L71-L92 for details.
+#[cfg(unix)]
+pub fn disable_rust_signal_handlers() -> Result<(), Errno> {
```

Comment:
could you please add a test for this?

Reply:
I added a test for the sleep utility, which relies on this function working as intended. I verified that the test fails if I revert my commit adding this function.

My intent was to not change more code than necessary, however I think kill() and kill_with_custom_signal() could be a single function. Same thing for try_kill() and try_kill_with_custom_signal.

Please also note that I'm a complete beginner in Rust.